### DISCUSSION TITLE
What about all pending things on the status page?

### DISCUSSION THREAD

**Discussion by Brosten** [dotnet/maui#4251] [General] [answered]
Looking at the status page, I see lots of things in status "pending" (= work not started). Also a lot in status "Underway".
https://github.com/dotnet/maui/wiki/Status
Examples:  

- Button.Command, Button.CommanParameter...
- Major part of carouselView
- Major part of collectionView
- Frame
- RadioButton
- WebViwe
- etc...

Looking at news in preview 12, I see almost no info around progress around the controls.
So, did you stop updating the status page, or is the status as bad as it looks? When shipping the release candidate, I would suspect all controls to be "green" on that page. But that doesn't seem resonable within two months, or?

Please enlighten me about how to read the status page correctly.

// Andreas

**Comment by Brosten** [comment]
I remember the early times of AppCenter. MS provided on site chat functionality letting you talk to people close to dev with almost no delay. It gave me the feeling of MS really making efforts in the area. They even called me and scheduled a meeting to interview me about my experience using AppCenter.

Now AppCenter is pretty much abandoned, Xamarin UI Test as well. Ok, well, probably they put all their efforts in making MAUI superb! Or...? In that case, why the silence? Why isn't a singel person put to follow the discussions here on GitHub? I see a lot of threads here that lacks simple replys from inside MS. 

Even if no defenitive answers can be given to some of the questiona, please at least start reacting and show the few of us still believing in Xamarin/Maui that you care...

**Comment by hartez** [comment, marked as answer]
We are still updating the status page (several updates in the last few days, in fact). Much of the Preview 12 work focused on internals rather than controls, and also the holidays slowed things down a bit. We hope to be updating more of the controls statuses in the next preview.

**Reply by Brosten** [reply]
Thanx for answering. Is the plan to complete it all before RC1?

Keep up the good qork! Just don't forget your community... 😉

**Reply by hartez** [reply]
The plan is to complete it. But things don't always go entirely to plan.
